custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1249,114 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+# performance row is AWCIncentiveReport object with 'awc_id', 'is_launched', 'valid_visits',
```

Comment:
Better to add this info as function doc instead of inline comment.

Reply:
okay